tutorials/parameter_sweep_demo.ipynb

```diff
@@ -0,0 +1,783 @@
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Appendix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a37bcaf",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Example Slurm Job submission script on NREL's Kestrel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "105c8406",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "### Slurm Batch File\n",
+    "\n",
+    "```bash\n",
+    "#!/bin/bash \n",
+    "#SBATCH --nodes=1  # Run the tasks on the same node\n",
+    "#SBATCH --ntasks-per-node=104 # Tasks per node to be run\n",
+    "#SBATCH --time=1:00:00   # Required, estimate 5 minutes\n",
+    "#SBATCH --account=hpcapps # Required\n",
+    "#SBATCH --partition=debug\n",
+    "#SBATCH --mail-user=kinshuk.panda@nrel.gov\n",
+    "#SBATCH --mail-type=BEGIN\n",
+    "#SBATCH --mail-type=END\n",
+    "#SBATCH --mail-type=FAIL\n",
+    "#SBATCH --mail-type=REQUEUE\n",
+    "\n",
+    "cd /home/kpanda/NAWI/watertap/tutorials/parameter_sweep_demo\n",
+    "module purge\n",
+    "module load craype-x86-spr\n",
+    "module load gcc/13.1.0 anaconda3/2022.05 netlib-lapack/3.11.0-gcc\n",
+    "conda activate /projects/hpcapps/kpanda/conda-envs/watertap\n",
+    "\n",
+    "mkdir -p outputs\n",
+    "N_SAMPLES=5000\n",
+    "NPROCS=100\n",
+    "\n",
+    "python parameter_sweep_demo_script.py $N_SAMPLES $NPROCS > outputs/fout_mp_${N_SAMPLES}_${NPROCS} 2> outputs/errout__mp_${N_SAMPLES}_${NPROCS}\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a934ae6",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "### Parameter Sweep Script"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40db8a69",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from IPython.display import Code\n",
+    "\n",
+    "Code(filename='assets_parameter_sweep_demo/parameter_sweep_demo_script.py', language=\"python\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "faeb4493-c2ea-46d0-8ca2-677c6d9b6464",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "celltoolbar": "Slideshow",
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.11.5"
+  },
+  "rise": {
+   "enable_chalkboard": true,
+   "height": "100%",
+   "scroll": true,
+   "slidenumber": true,
+   "theme": "simple"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
may be an empty cell at the end to delete

Reply:
Yep, removed.